Docs/Configuration.tex

```diff
@@ -112,31 +112,31 @@ \section{Introduction}\label{introduction}
 
 This document provides information on
 \href{https://github.com/acidanthera/OpenCorePkg}{OpenCore} user
-configuration file format used to setup the correct functioning of macOS
+configuration file format used to setup the correct functioning of the macOS
 operating system. It is to be read as the official clarification of expected
 OpenCore behaviour. All deviations, if found in published OpenCore releases,
-shall be considered documentation or implementation bugs, and are requested to be
-reported through \href{https://github.com/acidanthera/bugtracker}{Acidanthera Bugtracker}.
-Errata sheet is available in
+shall be considered to be documentation or implementation bugs which should be
```

Comment:
How about *bugs of documentation or implementation*?

Reply:
Whatever is more idiomatic, not a native speaker, but I thought "documentation or implementation bugs" is more common.

Reply:
Doesn't sound right

Reply:
Maybe *mistakes in documentation or bugs of implementation*? Sounds wordy though...

Reply:
In my opinion, leave it is as is.

Reply:
Right. Marked as resolved.